models/cam-fv/camdart_obs_preprocessor.f90

```diff
@@ -0,0 +1,320 @@
+! DART software - Copyright UCAR. This open source software is provided
+! by UCAR, "as is", without charge, subject to all terms of use at
+! http://www.image.ucar.edu/DAReS/DART/DART_download
+
+!> Simple program that opens an obs_seq file and loops over the observations
+!> and copies them to a new output file. This is intended to be a
+!> template for programs that want to alter existing obs in some simple way.
+!>
+!> NOTE:
+!> Observations in a sequence file are processed in ascending time order.
+!> For efficiency at creation time, sequence files support observations in a
+!> different physical order than temporal order.  This program will reorder
+!> the observations to be in temporal order, so the output observations
+!> may have a different observation index number than in the input file.
+
+program camdart_obs_preprocessor
```

Comment:
Consider changing name to be consistent with {mpas,wrf}_dart_obs_preprocessor
(separating model from "dart").
I don't mind working through the files to make all the changes.

Reply:
note to self: never rely on memory alone.  i thought this matched wrf and mpas, but no - you're right about the name.  if you're willing to make this change that would be great.  i agree it should be consistent with the others.